rust/makefile

```diff
@@ -52,6 +53,16 @@ target/universal/release/libexample.a: $(SOURCES) ndk-home
 	fi
 	@echo "[DONE] $@"
 
+## macos: Compile the macos libraries
+macos: target/x86_64-apple-darwin/release/libexample.dylib
+
+target/x86_64-apple-darwin/release/libexample.dylib: $(SOURCES) ndk-home
```

Comment:
Why only adding a target for x86_64 and not `aarch64`?
Why does the target depend on ndk-home (Android)?